Lib/sysconfig.py

```diff
@@ -216,6 +216,11 @@ def _expand_vars(scheme, vars):
     if vars is None:
         vars = {}
     _extend_dict(vars, get_config_vars())
+    if os.name == 'nt':
+        # On Windows we want to substitute 'lib' for schemes rather
+        # than the native value (without modifying vars, in case it
+        # was passed in)
+        vars = vars | {'platlibdir': 'lib'}
```

Comment:
Nice use of dict concatenation. 🙂

Reply:
I kind of want to change it to this:

```suggestion
        vars = {**vars, 'platlibdir': 'lib'}
```

Feels more readable. Every time I skim over this one I miss the `|`.